### DISCUSSION TITLE
Can we do login and signup without OpenID?

### DISCUSSION THREAD

**Discussion by awsomedesigner** [lowdefy/lowdefy#1180] [Questions and Help] [answered]
Can we do login and signup without OpenID? 

I want to create one app using lowdefy. I search over the internet and GitHub but didn't find any solution regarding this.

Can anyone elaborate more on this.

Thank you

**Comment by SamTolmay** [comment, marked as answer]
Hi @awsomedesigner 

In Lowdefy version 3 we only support OpenID Connect as an authentication provider. You can use a service like Auth0 or Okta to act as the provider. They can then also integrate with other third party providers like Google, or provide email magic links.

In Lowdefy version 4 we will are working on expanding the authentication system to allow for a lot more flexibility.
